### PR TITLE
osu!idol 2018 auditions announcement

### PR DESCRIPTION
**Pull request by Flanster**
### osu!idol 2018 announcement PR
Adding osu!idol 2018 auditions announcement.

### Status
Pending Review

### CODE REVIEW

**Comment by osuShiro** [comment]
announcement date is July first, please don't merge before that

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+date: 2018-07-01 00:00:00 +0000
+---
+
+May I interest you in some singing? Want to compete with others to become the next osu!idol? Then grab a mic and join in, it’s that time of year again to submit your entries and show us what you've got!
+
+![](/wiki/shared/news/2018-07-01-osu-idol-2018-auditions-now-open/banner.jpg)
+
+### Hold on, what is this?
+
+The annual community driven singing contest - osu!idol! It's where contestants as a solo or a duet sing their way through multiple stages of different genres with the aim to reach the very top of the idol podium! The panel of judges we have prepared will evaluate your performances and ensure the best is standing at the final stretch. After that, you, the community, will decide with a voting to determine our next osu!idol! It's go time!
```
Comment (osuShiro) [inline news/2018-07-01-osu-idol-2018-auditions-now-open.md]: "decide to determine" is heavy and doesn't ready well. Reword this with something like "After that, you, the community, can vote to determine who our next osu!idol is"

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![](/wiki/shared/news/2018-07-01-osu-idol-2018-auditions-now-open/banner.jpg)
+
+### Hold on, what is this?
+
+The annual community driven singing contest - osu!idol! It's where contestants as a solo or a duet sing their way through multiple stages of different genres with the aim to reach the very top of the idol podium! The panel of judges we have prepared will evaluate your performances and ensure the best is standing at the final stretch. After that, you, the community, will decide with a voting to determine our next osu!idol! It's go time! 
+
+### What should I do?
+
+All you need to do is make a small recording of around 30 seconds with your best performance without any background music. We'll be taking submissions up to a month so no theres no need to rush! Practice and make most of your only shot, otherwise you'd have to wait for next year!
```
Comment (osuShiro) [inline news/2018-07-01-osu-idol-2018-auditions-now-open.md]: "We'll be taking submissions up to a month" -> we'll be taking submissions for a month starting today

also "so no theres no need to rush" -> so there's no need to rush

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Did anyone say prizes?
+
+Top three will receive supporter tags with a bonus for the winner - the osu!idol winner badge!
```
Comment (osuShiro) [inline news/2018-07-01-osu-idol-2018-auditions-now-open.md]: Top three what ? After judge or community vote ? This needs clarification.

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* **1st:** osu!idol 2018 Winner profile Badge + 6 months supporter tag.
```
Comment (osuShiro) [inline news/2018-07-01-osu-idol-2018-auditions-now-open.md]: wasn't there a "osu!idol" title that was kept for a year as well ?

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md (hunk 1 above)
Comment (Flanster) [reply news/2018-07-01-osu-idol-2018-auditions-now-open.md]: Changed, but went with "who our next *osu!idol will be*"

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md (hunk 2 above)
Comment (Flanster) [reply news/2018-07-01-osu-idol-2018-auditions-now-open.md]: Applied

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md (hunk 4 above)
Comment (Flanster) [reply news/2018-07-01-osu-idol-2018-auditions-now-open.md]: osu!idol 2016 Winner, osu!idol 2017 Winner etc, it does not repeat with the same identical title.

File: news/2018-07-01-osu-idol-2018-auditions-now-open.md (hunk 3 above)
Comment (Flanster) [reply news/2018-07-01-osu-idol-2018-auditions-now-open.md]: clarified

**Pull request merged**